tests/lang-rust.yaml

```diff
@@ -2,24 +2,44 @@
   command: [rustc --version]
   entrypoint: [bash, -i, -c]
   assert:
-  - status == 0
-  - stdout.indexOf("rustc") != -1
+    - status == 0
+    - stdout.indexOf("rustc") != -1
 - desc: it should have properly configured rustup
   command: [rustup show]
   entrypoint: [bash, -i, -c]
   assert:
-  - status == 0
-  - stdout.indexOf("/home/gitpod/.rustup") != -1
-  - stdout.indexOf("x86_64-unknown-linux-gnu") != -1
-  - stdout.indexOf("x86_64-unknown-linux-musl") == -1
+    - status == 0
+    - stdout.indexOf("/home/gitpod/.rustup") != -1
+    - stdout.indexOf("x86_64-unknown-linux-gnu") != -1
+    - stdout.indexOf("x86_64-unknown-linux-musl") == -1
 - desc: it should have cargo
   command: [cargo --version]
   entrypoint: [bash, -i, -c]
   assert:
-  - status == 0
+    - status == 0
 # - desc: it should have matching rustc and cargo versions
 #   command: [cargo --version; rustc --version]
 #   entrypoint: [bash, -i, -c]
 #   assert:
 #   - status == 0
 #   - stdout.split("\n")[0].split(" ")[1] == stdout.split("\n")[1].split(" ")[1]
```

Comment:
This will not work because stdout can be read only once. AFAIK atm dazzle does not allow you to create a variable out of stdout/stderr.

Reply:
Umm, I didn't write that code block of `it should have matching rustc and cargo versions`.